doc-content/enterprise-only/weblogic/wls-configure-embedded-engine-proc.adoc

```diff
@@ -0,0 +1,286 @@
++
+.Client dependency
+[source,xml]
+----
+<dependency>
+  <groupId>org.uberfire</groupId>
+  <artifactId>uberfire-rest-client</artifactId>
+</dependency>
+----
++
+* When creating a JAR file that includes assets, such as rules and process definitions, specify the packaging type for your Maven project as `kjar` and use `org.kie:kie-maven-plugin` to process the `kjar` packaging type located under the `<project>` element. In the following example, `${kie.version}` is the Maven library version listed in
+ifdef::PAM[]
+https://access.redhat.com/solutions/3405361[What is the mapping between Red Hat Process Automation Manager and the Maven library version?]:
+endif::[]
+ifdef::DM[]
+https://access.redhat.com/solutions/3363991[What is the mapping between Red Hat Decision Manager and the Maven library version?]:
+endif::[]
++
+[source,xml]
+----
+<packaging>kjar</packaging>
+<build>
+ <plugins>
+  <plugin>
+   <groupId>org.kie</groupId>
+   <artifactId>kie-maven-plugin</artifactId>
+   <version>${kie.version}</version>
+   <extensions>true</extensions>
+  </plugin>
+ </plugins>
+</build>
+----
+
+
+ifdef::PAM[]
+. If the {PROCESS_ENGINE} or the {DECISION_ENGINE} along with persistence support is used in your project, you must declare the following hibernate dependencies in the `dependencyManagement` secion of your `pom.xml` file.
++
+[id='hibernate_dependencies_with_process_engine_and_decision_engine']
+.Hibernate dependencies
+[source,xml]
+----
+<!-- hibernate dependencies -->
+<dependencyManagement>
+  <dependencies>
+    <dependency>
+    <groupId>org.hibernate</groupId>
+    <artifactId>hibernate-entitymanager</artifactId>
+    <version>${version.org.hibernate-4ee7}</version>
+    </dependency>
+
+    <dependency>
+    <groupId>org.hibernate</groupId>
+    <artifactId>hibernate-core</artifactId>
+    <version>${version.org.hibernate-4ee7}</version>
+    </dependency>
+  </dependencies>
+</dependencyManagement>
+----
+endif::[]
++
+ifdef::DM[]
+. If the {DECISION_ENGINE} is used along with persistence support in your project, you must declare the following hibernate dependencies in the `dependencyManagement` secion of your `pom.xml` file.
++
+[id='hibernate_dependencies_with_decision_engine']
+.Hibernate dependencies in {DECISION_ENGINE} with persistence
+[source,xml]
+----
+<!-- hibernate dependencies -->
+<dependencyManagement>
+  <dependencies>
+    <dependency>
+    <groupId>org.hibernate</groupId>
+    <artifactId>hibernate-entitymanager</artifactId>
+    <version>${version.org.hibernate-4ee7}</version>
+    </dependency>
+
+    <dependency>
+    <groupId>org.hibernate</groupId>
+    <artifactId>hibernate-core</artifactId>
+    <version>${version.org.hibernate-4ee7}</version>
+    </dependency>
+  </dependencies>
+</dependencyManagement>
+----
+endif::[]
++
+[NOTE]
+====
+The current version of the property `version.org.hibernate-4ee7` should be retrieved from the {PRODUCT_BA} BOM Parent.
```

Comment:
Copy the current version of the  `version.org.hibernate-4ee7` property from the {PRODUCT_BA} BOM Parent.
What is the {PRODUCT_BA} BOM Parent?